src/platform/dbghelp_syminit_manager.hpp

```diff
@@ -1,42 +1,19 @@
 #ifndef DBGHELP_SYMINIT_MANAGER_HPP
 #define DBGHELP_SYMINIT_MANAGER_HPP
 
-#include "utils/common.hpp"
-#include "utils/utils.hpp"
-
 #include <unordered_set>
 
-#include <windows.h>
-#include <dbghelp.h>
-
 namespace cpptrace {
 namespace detail {
     struct dbghelp_syminit_manager {
-        std::unordered_set<HANDLE> set;
-
-        ~dbghelp_syminit_manager() {
-            for(auto handle : set) {
-                if(!SymCleanup(handle)) {
-                    ASSERT(false, microfmt::format("Cpptrace SymCleanup failed with code {}\n", GetLastError()).c_str());
-                }
-            }
-        }
+        std::unordered_set<void*> set;
```

Comment:
It'd be helpful to add a comment here indicating it's a `HANDLE`, just to make the intent clear

Reply:
Good point, will do!